colossus-testkit/src/main/scala/colossus-testkit/CallbackMatchers.scala

```diff
@@ -0,0 +1,40 @@
+package colossus
+package testkit
+
+import colossus.service.Callback
+import org.scalatest.matchers.{MatchResult, Matcher}
+
+import scala.util.{Failure, Success}
+
+trait CallbackMatchers {
+
+  class CallbackEvaluateTo[T](f : T => Any) extends Matcher[Callback[T]] {
```

Comment:
I plan to make a negative matcher, which will match on Failures.  This will be to assert exceptions, etc.